iml-api/src/graphql.rs

```diff
@@ -80,6 +80,23 @@ struct TargetResource {
     cluster_hosts: Vec<i32>,
 }
 
+#[derive(juniper::GraphQLObject)]
+/// A banned resource
+struct BannedResource {
+    /// The banned resource id
+    id: String,
+    /// The banned resource
+    resource: String,
+    /// The banned node
+    node: String,
+    /// The cluster id in which the banned resource lives
+    cluster_id: i32,
+    /// The host id for the banned resource
+    host_id: i32,
+    // The banned resource's mount point
+    mount_point: Option<String>,
```

Comment:
A banned resource is not guaranteed to be a target.

Reply:
Sure, should I rename it to `BannedTargetResource`?